createAccountFragment/src/main/res/layout/fragment_layout.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.fragment.app.FragmentContainerView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Make one reusable layout for this

Reply:
refactored for this. removed both module activities and it now lives in the app module.